src/jupyter_contrib_nbextensions/nbextensions/limit_output/readme.md

```diff
@@ -37,3 +37,13 @@ counted, and if either exceeds `limit_output` characters will be truncated to
 The `limit_output_message` parameter can be formatted to display the
 `limit_output` length and the current `output_length`, using the respective
 replacement fields `{limit_output_length}` and `{output_length}`.
+
+
+Parameter | Type | Description 
```

Comment:
Unfortunately table markdown isn't supported by the recommonmark parser which is currently producing the readthedocs pages. I've used bullet lists instead, although they aren't really as good as a table would be